README.md

```diff
@@ -1243,3 +1243,78 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+
+### Zwave Specific service 
+This service is used to meet all zwave specific requirements that are not satisfied with already implemented services.
+
+#### Service name 
+
+`zwave_specific`
+
+#### Interfaces
+
+Type | Interface               | Value type | Description 
+-----|-------------------------|------------|--------------
+out  | evt.notification.report | str_map    | Notification Reports that are sent by slave devices.
+
+#### Storage property
+
+Key        | Value description 
+-----------|----------------------------------
+sub_value  | Values pair to store in Vinculum.
```

Comment:
I don't get what `sub_value` is? Subscription value? Typo for supported value? And I don't get what the description means either :|

Reply:
It defines how values are stored in Vinculum persistant storage. It defines a pair of keys which should be treated as unique to hold as persistant data. 
For example now for every evt.notification.report Vinculum would keep only one value.
But I want a Vinculum to hold every value with unique pair of keys eg. "home_security:motion_sensor_status".
So in that way when another notification occurs (eg. home_security:cover_status), Vinculum will not replace previous one, but will keep both.